File: OpenCredentialPublisher.Data/ViewModels/nG/PackageVM.cs
using OpenCredentialPublisher.Data.Extensions;
using OpenCredentialPublisher.Data.Models;
using OpenCredentialPublisher.Data.Models.Enums;
using System;
using System.Collections.Generic;
using Legacy = OpenCredentialPublisher.Data.ViewModels.Credentials;

namespace OpenCredentialPublisher.Data.ViewModels.nG
{
    public class PackageVM
    {
        public int Id { get; set; }
        public PackageTypeEnum TypeId { get; set; }
        public int AssertionCount { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime ModifiedAt { get; set; }
        public bool IsOwner { get; set; }
        public bool ShowDownloadPdfButton { get; set; }
        public bool ShowDownloadVCJsonButton { get; set; }
        public List<int> ClrIds { get; set; }
        public PdfShareViewModel NewestPdfTranscript { get; set; }

        public PackageVM()
        {
            AssertionCount = 0;
            ClrIds = new List<int>();
        }
    }
}

File: OpenCredentialPublisher.Wallet/src/ClientApp/src/app/shared/models/packageVM.ts
import { PackageTypeEnum } from "./enums/packageTypeEnum";
import { PdfShare } from "./pdfShare";

export class PackageVM {
  id: number;
  isOwner: boolean;
  typeId: PackageTypeEnum;
  assertionCount: number;
  createdAt: Date;
  modifiedAt: Date;
  showDownloadPdfButton: boolean;
  showDownloadVCJsonButton: boolean;
  clrIds: Array<number>
  newestPdfTranscript: PdfShare;
  constructor () {
    this.id = 0;
    this.typeId = 0;
    this.assertionCount = 0;
    this.createdAt = null;
    this.modifiedAt = null;
    this.isOwner = false;
    this.showDownloadPdfButton = true;
    this.showDownloadVCJsonButton = true;
    this.clrIds = new Array<number>();
    this.newestPdfTranscript = null;
  }
}
